src/components/app/data/hooks/useEnterpriseCourseEnrollments.js

```diff
@@ -66,7 +69,11 @@ export default function useEnterpriseCourseEnrollments(queryOptions = {}) {
       ...queryEnterpriseCourseEnrollments(enterpriseCustomer.uuid),
       ...queryOptions,
       select: (data) => {
-        const transformedData = data.map(transformCourseEnrollment);
+        const enrollments = data.map(transformCourseEnrollment);
+        const transformedData = {
+          enrollments,
+          enrollmentsByStatus: groupCourseEnrollmentsByStatus(enrollments),
+        };
```

Comment:
[inform] Parity with the `transformedData` returned by the `bffQueryOptions` above, such that the result `data` from either the bff query or the fallback query has the same data structure.